src/app-layout/skeleton/layout.tsx

```diff
@@ -52,27 +50,20 @@ export function SkeletonLayout({
       style={{
         minBlockSize: `calc(100vh - ${placement.insetBlockStart}px - ${placement.insetBlockEnd}px)`,
         [customCssProps.maxContentWidth]: isMaxWidth ? '100%' : maxContentWidth ? `${maxContentWidth}px` : '',
-        [customCssProps.minContentWidth]: `${minContentWidth}px`,
+        [customCssProps.navigationWidth]: `${navigationWidth}px`,
+        [customCssProps.toolsWidth]: `${toolsWidth}px`,
       }}
     >
       <section className={styles['top-bar']}>{topBar}</section>
-      {navigation && (
-        <div className={styles.navigation} style={{ inlineSize: navigationWidth }}>
```

Comment:
Moved inline styles to a CSS-variable to allow conditionally turn it on and off using media query. Otherwise I would have to override this width on mobile 

```
@include styles.media-breakpoint-up(styles.$breakpoint-x-small) {
  inline-size: 100% !important;
}
```

which does not look good